aspnetcore/includes/mvc-intro/validation.md

```diff
@@ -18,13 +18,13 @@ Update the `Movie` class to take advantage of the built-in `Required`, `StringLe
 
 ::: moniker range=">= aspnetcore-2.1"
 
-[!code-csharp[](~/tutorials/first-mvc-app/start-mvc//sample/MvcMovie21/Models/MovieDateRatingDA.cs?name=snippet1)]
+[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie21/Models/MovieDateRatingDA.cs?name=snippet1)]
```

Comment:
I checked the references to the files I updated so I could do a sanity check. I couldn't find a few of the references because of the double `/` so I just fixed those up whilst I was looking.